### PR TITLE
refactor(Event-active-status.js): refactored the component in the eve…

### PR DESCRIPTION
**Pull request by drsckarasu**
…nt folder and change constants in eventStatusEnum.js

dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/event/EventActiveStatus/Event-active-status.js [hunk 1]
```diff
@@ -7,13 +7,14 @@ import eventStatusIcon from "../../../constants/eventStatusIcon";
 import SimpleModalWithDetails from "../../helpers/simple-modal-with-details";
 import RenderEventButton from "./RenderEventButton";
 
+const ARE_YOU_SURE = "Are you sure?";
```
Comment (VadymKharyton) [inline src/components/event/EventActiveStatus/Event-active-status.js]: Extract this constant variable into separate file with other constants

File: src/components/event/EventActiveStatus/RenderEventButton.js [hunk 2]
```diff
@@ -3,10 +3,11 @@ import IconButton from "@material-ui/core/IconButton";
 import Tooltip from "@material-ui/core/Tooltip";
 import propTypes from "prop-types";
 
+const MIDDLE = "middle";
```
Comment (VadymKharyton) [inline src/components/event/EventActiveStatus/RenderEventButton.js]: Extract this constant variable into separate file with other constants

File: src/components/helpers/form-helpers/render-date-pickerV2.jsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import {
   MuiPickersUtilsProvider,
   KeyboardDatePicker,
 } from "@material-ui/pickers";
-import DateFnsUtils from "@date-io/date-fns";
```
Comment (Stanislav-Vyhnan) [inline src/components/helpers/form-helpers/render-date-pickerV2.jsx]: It doesn't need to be changed. It must remain

**Review by Stanislav-Vyhnan — CHANGES_REQUESTED** [review]

File: src/components/event/EventActiveStatus/Event-active-status.js [hunk 4]
```diff
@@ -7,13 +7,14 @@ import eventStatusIcon from "../../../constants/eventStatusIcon";
...
 const EventActiveStatus = ({ eventStatus, eventId, onBlock, onUnBlock }) => {
   switch (eventStatus) {
-    case eventStatusEnum.Active:
+    case eventStatusEnum.ACTIVE:
```
Comment (andrii-kuzmichov) [inline src/components/event/EventActiveStatus/Event-active-status.js]: use destructuring for each cases

File: src/components/event/EventFilter/Event-filter.js [hunk 5]
```diff
@@ -79,9 +79,9 @@ class EventFilter extends Component {
 
     const values = formValues || { selectedPos: {} };
     const options = [
-      { value: eventStatusEnum.Active, text: OPTIONS_ACTIVE },
-      { value: eventStatusEnum.Blocked, text: OPTIONS_BLOCKED },
-      { value: eventStatusEnum.Canceled, text: OPTIONS_CANCELED },
+      { value: eventStatusEnum.ACTIVE, text: OPTIONS_ACTIVE },
+      { value: eventStatusEnum.BLOCKED, text: OPTIONS_BLOCKED },
+      { value: eventStatusEnum.CANCELED, text: OPTIONS_CANCELED },
```
Comment (andrii-kuzmichov) [inline src/components/event/EventFilter/Event-filter.js]: the same as above

File: src/components/event/EventItem/Event-item.js [hunk 6]
```diff
@@ -69,8 +69,8 @@ const EventCard = props => {
         className={classes.cardCanceled}
         style={{
           backgroundColor:
-            eventStatus === eventStatusEnum.Blocked ? "gold" : "",
-          opacity: eventStatus === eventStatusEnum.Canceled ? 0.5 : 1,
+            eventStatus === eventStatusEnum.BLOCKED ? "gold" : "",
+          opacity: eventStatus === eventStatusEnum.CANCELED ? 0.5 : 1,
```
Comment (andrii-kuzmichov) [inline src/components/event/EventItem/Event-item.js]: the same

Comment (andrii-kuzmichov) [reply src/components/event/EventItem/Event-item.js]: create constants for magic string and number like as `gold`, `0.5`, '1'

File: src/components/event/EventItemView/Event-item-view.js [hunk 7]
```diff
@@ -119,23 +119,23 @@ export default class EventItemView extends Component {
       isFreePlace &&
       !iWillVisitIt &&
       !isMyEvent &&
-      eventStatus === eventStatusEnum.Active &&
+      eventStatus === eventStatusEnum.ACTIVE &&
```
Comment (andrii-kuzmichov) [inline src/components/event/EventItemView/Event-item-view.js]: Use destructuring for each cases

File: src/components/event/EventList/Event-list.js [hunk 8]
```diff
@@ -65,9 +65,9 @@ const mapDispatchToProps = dispatch => {
     resetEvents: () => dispatch(resetEvents()),
     updateEventsFilters: filter => dispatch(updateEventsFilters(filter)),
     onBlock: (eventId, reason) =>
-      dispatch(changeEventStatus(eventId, reason, eventStatusEnum.Blocked)),
+      dispatch(changeEventStatus(eventId, reason, eventStatusEnum.BLOCKED)),
```
Comment (andrii-kuzmichov) [inline src/components/event/EventList/Event-list.js]: the same

File: src/containers/event-draft.js [hunk 9]
```diff
@@ -146,7 +146,7 @@ const mapDispatchToProps = dispatch => {
     editEvent: (data, onError, onSuccess) =>
       dispatch(editEvent(data, onError, onSuccess)),
     delete: (eventId, reason) =>
-      dispatch(changeEventStatus(eventId, reason, eventStatusEnum.Deleted)),
+      dispatch(changeEventStatus(eventId, reason, eventStatusEnum.DELETED)),
```
Comment (andrii-kuzmichov) [inline src/containers/event-draft.js]: the same

File: src/containers/event-edit-wrapper.js [hunk 10]
```diff
@@ -35,7 +35,7 @@ class EventEditWrapper extends Component {
 
     return (
       <SpinnerWrapper showContent={data !== undefined}>
-        {data.eventStatus === eventStatusEnum.Active ? (
+        {data.eventStatus === eventStatusEnum.ACTIVE ? (
```
Comment (andrii-kuzmichov) [inline src/containers/event-edit-wrapper.js]: the same

File: src/containers/event-item-view.js [hunk 11]
```diff
@@ -77,11 +77,11 @@ const mapDispatchToProps = dispatch => ({
...
   leave: (userId, eventId) => dispatch(leave(userId, eventId)),
   cancel: (eventId, reason) =>
-    dispatch(changeEventStatus(eventId, reason, eventStatusEnum.Canceled)),
+    dispatch(changeEventStatus(eventId, reason, eventStatusEnum.CANCELED)),
   unCancel: (eventId, reason) =>
-    dispatch(changeEventStatus(eventId, reason, eventStatusEnum.Active)),
+    dispatch(changeEventStatus(eventId, reason, eventStatusEnum.ACTIVE)),
   delete: (eventId, reason) =>
-    dispatch(changeEventStatus(eventId, reason, eventStatusEnum.Deleted)),
+    dispatch(changeEventStatus(eventId, reason, eventStatusEnum.DELETED)),
```
Comment (andrii-kuzmichov) [inline src/containers/event-item-view.js]: the same

File: src/components/helpers/form-helpers/render-date-pickerV2.jsx (hunk 3 above)
Comment (drsckarasu) [reply src/components/helpers/form-helpers/render-date-pickerV2.jsx]: done

File: src/components/event/EventActiveStatus/Event-active-status.js (hunk 4 above)
Comment (drsckarasu) [reply src/components/event/EventActiveStatus/Event-active-status.js]: done

File: src/components/event/EventActiveStatus/Event-active-status.js (hunk 1 above)
Comment (drsckarasu) [reply src/components/event/EventActiveStatus/Event-active-status.js]: done

File: src/components/event/EventActiveStatus/RenderEventButton.js (hunk 2 above)
Comment (drsckarasu) [reply src/components/event/EventActiveStatus/RenderEventButton.js]: done

File: src/components/event/EventFilter/Event-filter.js (hunk 5 above)
Comment (drsckarasu) [reply src/components/event/EventFilter/Event-filter.js]: done

File: src/components/event/EventItem/Event-item.js (hunk 6 above)
Comment (drsckarasu) [reply src/components/event/EventItem/Event-item.js]: done

File: src/components/event/EventItemView/Event-item-view.js (hunk 7 above)
Comment (drsckarasu) [reply src/components/event/EventItemView/Event-item-view.js]: done

File: src/components/event/EventList/Event-list.js (hunk 8 above)
Comment (drsckarasu) [reply src/components/event/EventList/Event-list.js]: done

File: src/containers/event-draft.js (hunk 9 above)
Comment (drsckarasu) [reply src/containers/event-draft.js]: done

File: src/containers/event-edit-wrapper.js (hunk 10 above)
Comment (drsckarasu) [reply src/containers/event-edit-wrapper.js]: done

File: src/containers/event-item-view.js (hunk 11 above)
Comment (drsckarasu) [reply src/containers/event-item-view.js]: done

File: src/components/event/EventItem/Event-item.js [hunk 12]
```diff
@@ -68,9 +69,11 @@ const EventCard = props => {
       <Card
         className={classes.cardCanceled}
         style={{
-          backgroundColor:
-            eventStatus === eventStatusEnum.Blocked ? "gold" : "",
-          opacity: eventStatus === eventStatusEnum.Canceled ? 0.5 : 1,
+          backgroundColor: eventStatus === BLOCKED ? eventItem.GOLD : "",
```
Comment (VadymKharyton) [inline src/components/event/EventItem/Event-item.js:72]: Replace with `eventStatus === BLOCKED && eventItem.GOLD`

File: .eslintrc [hunk 13]
```diff
@@ -13,7 +13,7 @@
     "import"
   ],
   "env": {
-    "browser": true
+    "browser": true // browser environment
```
Comment (VadymKharyton) [inline .eslintrc:16]: Remove this comment

**Review by VadymKharyton — COMMENTED** [review]
Fix these very small comments and PR will be approved

**Pull request closed without merging**